test/powershell/Modules/Microsoft.PowerShell.Utility/Send-MailMessage.Tests.ps1

```diff
@@ -0,0 +1,135 @@
+Describe "Basic Send-MailMessage tests" -Tags CI {
+    function read-mail
+    {
+        Param(
+            [parameter(Mandatory=$true)]
+            [String]
+            $mailBox
+        )
+
+        $state = "init"
+        $mail = Get-Content $mailBox
+        $rv = @{}
+        foreach ($line in $mail)
+        {
+            switch ($state)
+            {
+                "init"
+                {
+                    if ($line.Length -gt 0)
+                    {
+                        $state = "headers"
+                    }
+                }
+                "headers"
+                {
+                    if ($line.StartsWith("From: "))
+                    {
+                        $rv.From = $line.Substring(6)
+                    }
+                    elseif ($line.StartsWith("To: "))
+                    {
+                        if ($rv.To -eq $null)
+                        {
+                            $rv.To = @()
+                        }
+
+                        $rv.To += $line.Substring(4)
+                    }
+                    elseif ($line.StartsWith("Subject: "))
+                    {
+                        $rv.Subject = $line.Substring(9);
+                    }
+                    elseif ($line.Length -eq 0)
+                    {
+                        $state = "body"
+                    }
+                }
+                "body"
+                {
+                    if ($line.Length -eq 0)
+                    {
+                        $state = "done"
+                        continue
+                    }
+
+                    if ($rv.Body -eq $null)
+                    {
+                        $rv.Body = @()
+                    }
+
+                    $rv.Body += $line
+                }
+            }
+        }
+
+        return $rv
+    }
+
+    BeforeAll {
+        $PesterArgs = @{ Name = "Can send mail message from user to self"}
+        $alreadyHasMail = $true
+
+        if (-not $IsLinux)
+        {
+            $PesterArgs["Skip"] = $true
+            $PesterArgs["Name"] += " (skipped: not Linux)"
+            return
+        }
+
+        $user = "jeff"
+        $inPassword = Select-String "^${user}:" /etc/passwd -ErrorAction SilentlyContinue
+        if (-not $inPassword)
+        {
+            $PesterArgs["Pending"] = $true
+            $PesterArgs["Name"] += " (pending: user not in /etc/passwd)"
+            return
+        }
+
+        $domain = "Chinstrap"
```

Comment:
similar to above, this will almost always fail and thus be `Pending`, seems like if `mail` is installed you should be able to rely on root@computername?